test_instafail.py

```diff
@@ -31,17 +34,25 @@ def n(request):
     }[request.param]
 
 
-@pytest.fixture(params=['default', 'verbose', 'quiet'])
+@pytest.fixture(params=['default', 'verbose', 'quiet', 'ini'])
 def option(request, n):
     return {
         "default": Option(verbose=False, n=n),
         "verbose": Option(verbose=True, n=n),
         "quiet": Option(quiet=True, n=n),
+        "ini": Option(ini=True, n=n),
     }[request.param]
 
 
 class TestInstafailingTerminalReporter(object):
     def test_fail(self, testdir, option):
+        if option.ini:
```

Comment:
This is repeated a lot. How about this:

- Add an `instafail_option` fixture that's parametrized as either `False` or `True`.
- Add an autoused `ini` fixture that makes this ini file, if `instafail_option` is true
- Replace the `Option.ini` attribute with an `instafail` attribute. Pass the `instafail_option` fixture to `Option` constructors in the `option` fixture.